test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -813,13 +816,20 @@ class TestDapp {
   }
 
   async clickERC20TokenTransferButton() {
+    await this.driver.waitForSelector(this.erc20TokenTransferButton, {
+      state: 'enabled',
+    });
     await this.driver.clickElement(this.erc20TokenTransferButton);
   }
 
   async clickERC20WatchAssetButton() {
     await this.driver.clickElement(this.erc20WatchAssetButton);
   }
 
+  async clickERC20IncreaseAllowanceButton() {
+    await this.driver.clickElement(this.erc20IncreaseTokensAllowanceButton);
+  }
+
```

Comment:
same, could we add this method above to preserve alphabetical order if possible :pray: